pandas/core/resample.py

```diff
@@ -480,6 +480,21 @@ def backfill(self, limit=None):
         return self._upsample('backfill', limit=limit)
     bfill = backfill
 
+    def nearest(self, limit=None):
+        """
+        Backward or forward fill values
```

Comment:
This is not particularly descriptive, at least to me.  Also, add a newline beneath this one.

Reply:
i copy and pasted the structure from surrounding code

Reply:
Well then, take advantage of this opportunity to improve upon it :smile: